.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,7 +1,9 @@
 name: Probleem melden / Bug report
 description: Meldt een probleem om ons te helpen verbeteren  / Create a report to help us improve
 title: "Title here"
-labels: ["bug", "triage"]
+type: "Bug"
+labels: ["triage"]
```

Comment:
I personally filter on labels in the project board, so I'd keep "bug" label for this purpose

Reply:
hmm good point, I'll see if I can add the issue type to the display in the project board as well, that way you can do the same filtering but via the `Bug` issue type instead of the label :)

Reply:
I added the types to the project board display, currently these types aren't assigned yet for repos other than OZ/ON, but I think I'll have this done sometime next week.

I'll leave the bug and enhancement labels in place for now, sometime in the future we might be able to remove them, but there's no rush to that imo

Reply:
Sounds good!